buildSrc/assets/themes/chuunibyou/rikka/dark/rika.dark.chrome.definition.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "5412c41d-f76b-4488-85a7-1ae1a63bbfcc",
+  "overrides": {
+    "theme": {
+      "colors": {},
+      "properties": {
+        "ntp_background_alignment": "right"
```

Comment:
FYI: Rikka's background anchoring is center, because she is in the center of the asset.